src/backend/pool.go

```diff
@@ -43,6 +43,43 @@ var (
 	errClosed   = errors.New("can't get connection from the closed DB")
 )
 
+// Poolz ...
```

Comment:
Simply explain about Poolz:
Add replica and normal pool to distribute SQL between read and write in some cases for load-balanced.